classes/Bili/Date.php

```diff
@@ -33,114 +36,138 @@
 
 class Date
 {
-    public static function fromMysql($strFormat, $strDateTime)
+    /**
+     * Transform mysql format datetime string to given iso format.
+     *
+     * @param string $strIsoFormat
+     * @param string $strDateTime
+     * @return string
+     */
+    public static function fromMysql(string $strIsoFormat, string $strDateTime): string
     {
-        $strReturn = $strDateTime;
-
-        if ($strDateTime != "0000-00-00 00:00:00" && !empty($strDateTime)) {
-            $strTStamp = strtotime($strDateTime);
-
-            if ($strTStamp !== -1 || $strTStamp !== false) {
-                $strReturn = strftime($strFormat, $strTStamp);
-            }
+        Carbon::setLocale('auto');
+        if ($strDateTime !== "0000-00-00 00:00:00" && !empty($strDateTime)) {
+            $objCarbon = Carbon::createFromFormat('Y-m-d H:i:s', $strDateTime);
+            $strReturn = $objCarbon->isoFormat($strIsoFormat);
         } else {
             $strReturn = "";
         }
 
         return $strReturn;
     }
 
-    public static function toMysql($strDateTime = "")
+    /**
+     * Gives mysql string.
+     *
+     * @param string $strDateTime
+     * @return string
+     */
+    public static function toMysql(string $strDateTime = ""): string
     {
         $strReturn = $strDateTime;
-        $strFormat = "%Y-%m-%d %H:%M:%S";
+        $strFormat = "Y-m-d H:i:s";
 
         if (empty($strDateTime)) {
-            $strTStamp = strtotime("now");
+            $strTStamp = time();
         } elseif (is_numeric($strDateTime)) {
             $strTStamp = $strDateTime;
         } else {
             $strTStamp = strtotime($strDateTime);
         }
 
         if ($strTStamp !== -1 || $strTStamp !== false) {
-            $strReturn = strftime($strFormat, $strTStamp);
+            $strReturn = DateTime::createFromFormat('U', $strTStamp)->format($strFormat);
```

Comment:
You probably need to use `Carbon` here to parse the timestamp and use the ISO format.